doc/release-notes/5.0-release-notes.md

```diff
@@ -0,0 +1,321 @@
+
+Add the below JVM options beneath the -Ddataverse settings:  
+
+   <jvm-options>-Dfish.payara.classloading.delegate=false</jvm-options>
+   <jvm-options>-XX:+UseG1GC</jvm-options>
+   <jvm-options>-XX:+UseStringDeduplication</jvm-options>
+   <jvm-options>-XX:+DisableExplicitGC</jvm-options>
+
+9. Change any full pathnames /usr/local/glassfish4/... to /usr/local/payara5/... or whatever it is in your case. (Specifically check the -Ddataverse.files.directory and -Ddataverse.files.file.directory JVM options)
+
+10. In domain1/config/jhove.conf, change the hard-coded /usr/local/glassfish4 path, as above.
+
+(Optional): If you renamed your service account from glassfish to payara or appserver, update the ownership permissions. The Installation Guide recommends a service account of `dataverse`:
+
+   `sudo chown -R dataverse /usr/local/payara5/glassfish/domains/domain1`
+   `sudo chown -R dataverse /usr/local/payara5/glassfish/lib`
+
+11. You will also need to check that the service account has write permission on the files directory, if they are located outside the old Glassfish domain. And/or make sure the service account has the correct AWS credentials, if you are using S3 for storage.
+
+12. Finally, start Payara:
+
+   `sudo -u dataverse /usr/local/payara5/bin/asadmin start-domain`
+
+13. Deploy the Dataverse 5 warfile:
+
+   `sudo -u dataverse /usr/local/payara5/bin/asadmin deploy /path/to/dataverse-5.0.war`
+
+14. Then restart Payara:
+
+   `sudo -u dataverse /usr/local/payara5/bin/asadmin stop-domain`
+   `sudo -u dataverse /usr/local/payara5/bin/asadmin start-domain`
+
+15. Update Astrophysics Metadata Block (if used)
+
+   `wget https://github.com/IQSS/dataverse/releases/download/5.0/astrophysics.tsv`
+   `curl http://localhost:8080/api/admin/datasetfield/load -X POST --data-binary @astrophysics.tsv -H "Content-type: text/tab-separated-values"`
+
+16. (Recommended) Run ReExportall to update JSON Exports  
+
+   <http://guides.dataverse.org/en/5.0/admin/metadataexport.html?highlight=export#batch-exports-through-the-api>
+
+17. For installations using DataCite, pre-register DOIs:
+
+   `/api/pids/unreserved`  will report the ids of the datasets
+   `/api/pids/:persistentId/reserve` reserves the assigned DOI with DataCite (will need to be run on every id reported by the the first API).
+
+   Scripted, the whole process would look as follows (adjust as needed):
+
+```
+   API_TOKEN='xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx'
+
+   curl -s -H "X-Dataverse-key:$API_TOKEN" http://localhost:8080/api/pids/unreserved |
+   # the API outputs JSON; note the use of jq to parse it:
+   jq '.data.count[].pid' | tr -d '"' | 
+   while read doi
+   do
+      curl -s -H "X-Dataverse-key:$API_TOKEN" -X POST http://localhost:8080/api/pids/:persistentId/reserve?persistentId=$doi
+   done
+```
+
+18. Set up Standalone Zipper Service if desired. Please consult the scripts/zipdownload/README.md in the Dataverse 5 source tree for more information. 
```

Comment:
similarly, this is already explained in its own section above - not sure it needs to be repeated in the upgrade-to-payara instruction - ?

Reply:
I added the experimental note in https://github.com/IQSS/dataverse/commit/abc4d9bc136d027862dab7b8b72381b89e340d9a. I think it's fine to add this in two different instruction formats and it matches what we've done in previous notes but if you feel strongly I can revisit. I don't do the upgrades myself so if this sort of thing is not helpful to mention multiple times I'm fine to adjust it.